stories/amo/HeroRecommendation.js

```diff
@@ -46,6 +48,19 @@ storiesOf('HeroRecommendation', module)
             title: 'without image',
             sectionFn: () => render({ featuredImage: null }),
           },
+          {
+            title: 'with error',
+            sectionFn: () =>
+              render(
+                {},
+                {
+                  errorHandler: new ErrorHandler({
+                    id: 'some-id',
+                    capturedError: new Error('Some error'),
```

Comment:
> This doesn't work for some reason, and ends up generating an error from the `ErrorList` component when it tries to render it. I'm not sure why because, AFAICT it's doing the exact same thing as `createStubErrorHandler`, which I had trouble using here because `sinon` doesn't exist.

Stories use a real store so I suggest dispatching a real error. Example of usage with a real store:

https://github.com/mozilla/addons-frontend/blob/b16c8f80d5a57c8ce8d84c7a269e17834bedfb53/src/core/api/index.js#L214

Reply:
I did some work which has removed the error that was being generated by Storybook, but this still isn't working as more work needs to be done to enable a story to create a `store` and then be able to pass that `store` into the `Provider`. @kumar303 and I discussed this a bit on IRC. In order to land this in a timely fashion, I'm going to file a follow-up to get the story working for a version of the `HeroRecommendation` that displays an error.

Reply:
I filed #8777